website/docs/r/vm.html.markdown

```diff
@@ -12,17 +12,19 @@ Creates and manages a Virtualbox VM
 ## Example Usage
 
 ```hcl
-resource "virtualbox_vm" "node" {
-  count     = 2
-  name      = format("node-%02d", count.index + 1)
-  image     = "https://app.vagrantup.com/ubuntu/boxes/bionic64/versions/20180903.0.0/providers/virtualbox.box"
-  cpus      = 2
+resource "virtualbox_vm" "vm1" {
+  name   = "debian-11"
+  image  = "https://app.vagrantup.com/shekeriev/boxes/debian-11/versions/0.2/providers/virtualbox.box"
```

Comment:
I know that those images might be stale that were already in the doc, but do you mind pointing the examples to non-personal VMs but rather official public images?